### PR TITLE
Refs #10086: Try multiple gpg sites when installing rvm.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: templates/install_rvm.sh.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+USER=<%= @user %>
+
+SERVERS=(
+    hkp://keys.gnupg.net
+    hkp://pgp.mit.edu
+)
+
+for server in "${SERVERS[@]}"
+do
+    su -c "gpg --keyserver ${server} --recv-keys D39DC0E3" - $USER
```
Comment (stbenjam) [inline templates/install_rvm.sh.erb]: You could also just include the GPG key itself if there's some kind of network problem that's preventing the downloading.  Up to you.  

(RVM's gpg signing is still of dubious value anyway if the script we're getting from `https://get.rvm.io` isn't signed itself. Nothing's stopping the @TheNSABackdoorCommittee from doing a MITM attack there.)

**Comment by stbenjam** [comment]
APT, looks like just lint style stuff.

**Comment by jlsherrill** [comment]
Tested and worked fine for me APT too

File: templates/install_rvm.sh.erb (hunk 1 above)
Comment (ehelms) [inline templates/install_rvm.sh.erb]: What started this was the first server being unreachable. So if the key
ever changes we'd have to update whereas this should hopefully break never
again with the redundancy.

**Pull request merged**